src/corona_hakab_model/common/__init__.py

```diff
@@ -0,0 +1,10 @@
+__all__ = [
```

Comment:
why do we need this?

Reply:
i though we might wanna use it some times, it allows us to do 
`from common import *`
however we are not using it now, so i will remove it

Reply:
oh that's what it does, i didn't know